src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -164,8 +166,8 @@
   word-wrap: break-word;
   background-color: var(--pf-c-clipboard-copy__expandable-content--BackgroundColor);
   background-clip: padding-box;
-  border: var(--pf-c-clipboard-copy__expandable-content--BorderWidth) solid transparent;
-  outline-offset: var(--pf-c-clipboard-copy__expandable-content--OutlineOffset);
+  border: solid var(--pf-c-clipboard-copy__expandable-content--BorderColor);
```

Comment:
wanna make `border-style`  a variable too?

Reply:
Nope! We typically don't. Do you think we should?

```
src/patternfly//components/FileUpload/file-upload.scss:      border: var(--pf-c-file-upload--m-drag-hover--before--BorderWidth) solid var(--pf-c-file-upload--m-drag-hover--before--BorderColor);
src/patternfly//components/FileUpload/file-upload.scss:        border: var(--pf-c-file-upload__file-select__c-button--m-control--disabled--after--BorderWidth) solid;
src/patternfly//components/Dropdown/dropdown.scss:    border: var(--pf-c-dropdown__toggle--BorderWidth) solid;
src/patternfly//components/Dropdown/dropdown.scss:  border: var(--pf-c-dropdown__menu--BorderWidth) solid transparent;
src/patternfly//components/ContextSelector/context-selector.scss:    border: var(--pf-c-context-selector__toggle--BorderWidth) solid;
src/patternfly//components/ContextSelector/context-selector.scss:  border: var(--pf-c-context-selector__menu--BorderWidth) solid transparent;
src/patternfly//components/ContextSelector/context-selector.scss:  border-bottom: var(--pf-c-context-selector__menu-search--BorderBottomWidth) solid var(--pf-c-context-selector__menu-search--BorderBottomColor);
src/patternfly//components/OptionsMenu/options-menu.scss:    border: var(--pf-c-options-menu__toggle--BorderWidth) solid;
src/patternfly//components/OptionsMenu/options-menu.scss:  border: var(--pf-c-options-menu__menu--BorderWidth) solid transparent; // this is for high contrast mode in windows
...
```